code_generator/GdkCodeGenerator/GdkCodeGenerator.csproj

```diff
@@ -39,6 +39,7 @@
     <T4Files Include="Templates\UnityEnumContent.tt" />
     <T4Files Include="Templates\UnityEventGenerator.tt" />
     <T4Files Include="Templates\UnityGameObjectComponentDispatcherGenerator.tt" />
+    <T4Files Include="Templates\UnityReaderWriterGenerator.tt" />
```

Comment:
It would be good to have a generic name for things that can be injected. ReaderWriter is a little awkward and we don't expect these to the only two long term.

Reply:
Do you have a suggestion? `UnityComponentAccessorGenerator`?

Reply:
If we introduced something that is not a ReaderWriter but can be injected, wouldn't it more likely be in a different template anyway?

Reply:
Fair enough that might just be different. In which case lets not generalise anything yet and just solve this problem.